File: Python/Level-2(string)/vowels_and_consonants.py
str = input()
vowels = 0
consonants = 0

str_lower = str.lower()

for i in range(len(str)):
    if str[i] == 'a' or str[i] == 'e' or str[i] == 'i' or str[i] == 'o' or str[i] == 'u':
        vowels += 1
    else:
        consonants += 1

print(f"The no of vowels = {vowels}")
print(f"The no of consonants = {consonants}")

File: C/Level-2(string)/vowels_and_consonants.c
#include <ctype.h>
#include <stdio.h>

void vowels_and_consonants(const char str[])
{
    int vowel = 0;
    int consonant = 0;

    for (int i = 0; str[i] != '\0'; i++)
    {
        char ch = tolower((unsigned char)str[i]);

        if (isalpha((unsigned char)str[i]))
        {
            if (ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u')
            {
                vowel++;
            }
            else
            {
                consonant++;
            }
        }
    }

    printf("the no of vowels are %d\n", vowel);
    printf("the no of consonants are %d\n", consonant);
}

int main(void)
{
    char str[100];

    if (scanf("%99s", str) == 1)
    {
        vowels_and_consonants(str);
    }

    return 0;
}